app/views/dashboard/protocols/_protocols_list.html.haml

```diff
@@ -73,7 +73,8 @@
             = protocol.id
         %td.title
           = short_title_display protocol
-          = display_rmid_validated_protocol(protocol, t('protocols.summary.rmid_short_title'))
+          - if RESEARCH_MASTER_ENABLED
```

Comment:
Since `if RESEARCH_MASTER_ENABLED` was added to `display_rmid_validated_protocol`, could we remove all of these condition statements in the views? It would remove a lot of duplication.